.changeset/sharp-rockets-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-components": minor
```

Comment:
@grigasp Should this be `major`? Because `presentation-components` now requires `ThemeProvider` from `itwinui-react@3.0`?

Reply:
Yes, let's set this as major